packages/report-server/src/reportBuilder/output/types.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { Row, PeriodMetadata } from '../types';
+
+export interface Response extends PeriodMetadata {
+  results: Row[];
+}
```

Comment:
Decided to now return the period metadata along with results here. Should be reminiscent of the metadata that is returned from data-broker itself when we fetch analytics/events.

I believe it also returns a `dataElementCodesToNames` payload, so I will add that to the typings...